01-node-tutorial/answers/08-os-module.js

```diff
@@ -0,0 +1,16 @@
+const os = require('os')
+
+// info about current user
+const user = os.userInfo()
+console.log(user)
+
+// method returns the system uptime in seconds
+console.log(`The System Uptime is ${os.uptime()} seconds`)
+
+const currentOS = {
+  name: os.type(),
+  release: os.release(),
+  totalMem: os.totalmem(),
+  freeMem: os.freemem(),
+}
```

Comment:
Extra challenge (optional): look at the [node.js os module documentation](https://nodejs.org/api/os.html) and add one more piece of information about your machine here.